### PR TITLE
ncm-network: fix typos and updated comments

### PR DESCRIPTION
**Pull request by aka7**
Describe the change you are making here, in particular we would like to know:

changes to nmstate.pm to address comments in previous pr.

* Why the change is necessary.
- fixed typos and updated comments and docs to be more meaningful.
- commented out start_openvswitch, support for this will come later.

* What backwards incompatibility it may introduce.
none

### CODE REVIEW

**Comment by jouvin** [comment]
I'm on holidays, don't wait a feedback from me...

File: ncm-network/src/main/perl/nmstate.pm [hunk 1]
```diff
@@ -594,8 +602,8 @@ sub Configure
     #
     $self->enable_network_service();
 
-    # TODO: not tested with nmstate. leaving it here. needs work.
-    $self->start_openvswitch($ifaces, $ifup);
+    # TODO: openvswitch configs for nmstate. Commentted out for now.
```
Comment (jrha) [inline ncm-network/src/main/perl/nmstate.pm:605]:
```suggestion
    # TODO: openvswitch configs for nmstate. Commented out for now.
```

Comment (aka7) [reply ncm-network/src/main/perl/nmstate.pm]: sorry about this, done.

File: ncm-network/pom.xml [hunk 2]
```diff
@@ -4,7 +4,7 @@
   <groupId>org.quattor.cfg.module</groupId>
   <artifactId>network</artifactId>
   <packaging>pom</packaging>
-  <version>23.6.0-rc2-SNAPSHOT</version>
+  <version>21.12.1-ms6</version>
```
Comment (jrha) [inline ncm-network/pom.xml:7]: Revert this please.

**Pull request merged**